### PR TITLE
Fix-adminpage-padding

### PR DESCRIPTION
**Pull request by bertacsevercan**
- Add the same paddings to wrapper divs in the Admin page.

### CODE REVIEW

File: src/components/CouponsChart/style.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.couponsChart {
+  padding: 1rem 0;
+}
```
Comment (halitbatur) [inline src/components/CouponsChart/style.css:3]: new line

File: src/components/CreateCoupons/style.css [hunk 2]
```diff
@@ -24,8 +24,8 @@
   background-color: #e17007;
   color: white;
 }
-.coupons {
```
Comment (halitbatur) [inline src/components/CreateCoupons/style.css:27]: new line

**Review by halitbatur — CHANGES_REQUESTED** [review]

**Review by halitbatur — APPROVED** [review]
fix conflicts and its ready to be merged

**Review by ihab-sensei — APPROVED** [review]
Looks good!

**Pull request merged**